src/views/PancakeSquad/components/Buttons/ActivateProfile.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import { Link } from 'react-router-dom'
+import { Button } from '@pancakeswap/uikit'
+import { ContextApi } from 'contexts/Localization/types'
+import { UserStatusEnum } from 'views/PancakeSquad/types'
+
+type ActivateProfileButtonProps = {
+  t: ContextApi['t']
+  userStatus: UserStatusEnum
+}
+
+const ActivateProfileButton: React.FC<ActivateProfileButtonProps> = ({ t, userStatus }) => (
+  <>
+    {(userStatus === UserStatusEnum.NO_PROFILE || userStatus === UserStatusEnum.UNCONNECTED) && (
```

Comment:
I'd personally prefer simple
```jsx
if (userStatus === UserStatusEnum.NO_PROFILE || userStatus === UserStatusEnum.UNCONNECTED) {
  return (<Button as={Link} to="/create-profile" mr="4px">
        {t('Activate Profile')}
   </Button>)
}
return null
```
but its just a nitpick of course :)

Reply:
Do we have a convention in the team on that ? If not we could decide on one.